src/main/java/com/minecolonies/api/crafting/ItemStorage.java

```diff
@@ -46,7 +47,7 @@ public class ItemStorage
      */
     public ItemStorage(@NotNull final ItemStack stack, final int amount, final boolean ignoreDamageValue)
     {
-        this.stack = stack;
+        this.stack = stack.copyWithCount(1);
```

Comment:
we probably should clean up the callsites which are making unnecessary copies then

Reply:
Hmm, ok, I've removed those.  Haven't tested it since my test colony doesn't have the problematic recipe persisted any more, but I believe it should still work.